opentok/opentok.py

```diff
@@ -414,11 +424,17 @@ def create_session(
 
     def headers(self):
         """For internal use."""
+        user_agent = (
+            "OpenTok-Python-SDK/" + __version__ + " python/" + platform.python_version()
+        )
+        """ For user purposes. Append app name and version """
+        if self._app_name and self._app_version:
+            user_agent += " {app_name}/{app_version}".format(
+                app_name=self._app_name, app_version=self._app_version
+            )
```

Comment:
```suggestion
            user_agent += " {app_name}".format(
                app_name=self._app_name
            )
```

Works with the above edit, the `app_version` passed into the constructor is the SDK version, not a custom version we track via the 3rd chunk. The user is allowed to specify that 3rd chunk however they see fit.